### DISCUSSION TITLE
Display numeric values in tiles

### DISCUSSION THREAD

**Discussion by sstewart-hirez** [marcusolsson/grafana-treemap-panel#44] [Ideas]
Ability to display numeric values in tiles similar to this https://finviz.com/map.ashx

![image](https://user-images.githubusercontent.com/105306509/200370983-f164f761-1f9f-4bd5-8343-53ba9a265fed.png)

Based on this Slack conversation: https://grafana.slack.com/archives/C3HJV5PNE/p1666989999513239?thread_ts=1666900307.592699&cid=C3HJV5PNE

**Comment by tolzhabayev** [comment]
fyi @darrenjaneczek

**Comment by pncgithub** [comment]
Actually, i was looking for that kind of  feature, and playing with the additional labels, but ...they have another purpose.

**Comment by hoffmanroni** [comment]
Hi is there anyone actively working on this plugin?  Wondering about adding some kind of label into the treemap would be possible and seems it would be very useful.
